elfreloc_amd64.c

```diff
@@ -26,6 +26,9 @@ el_status el_applyrela(el_ctx *ctx, Elf_RelA *rel)
     uint32_t type = ELF_R_TYPE(rel->r_info);
     uint32_t sym  = ELF_R_SYM(rel->r_info);
 
+    (void) sym;
```

Comment:
Probably better to just remove the definition of sym here. Also, does the same change apply to any of the other architectures?

Reply:
I left it there in case there were some plans to use it later on. Still not sure what it's purpose is but it's used in the other architectures.